packages/core/src/tracing/cls.js

```diff
@@ -545,20 +545,28 @@ function call(fn) {
  *
  * @param {Object.<string, *>} options
  */
-function skipExitTracing(
-  options = {
+function skipExitTracing(options) {
+  const opts = Object.assign({}, options, {
     isActive: true,
     extendedResponse: false,
     skipParentSpanCheck: false,
     log: true,
     skipIsTracing: false
-  }
-) {
+  });
+
```

Comment:
_issue(blocking):_ Oh right, the previous code was definitely broken. Good catch. However, this also does not work correctly, I think? See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign?retiredLocale=de#merging_objects_with_same_properties ->

> The properties are overwritten by other objects that have the same properties later in the parameters order.

I think we need to put the defaults first and the actual incoming parameters last, so they overwrite the defaults. I think what you want is this:

```
function skipExitTracing(options) {
  const opts = Object.assign(
    {
      isActive: true,
      extendedResponse: false,
      skipParentSpanCheck: false,
      log: true,
      skipIsTracing: false
    },
    options
  );
  // console.table(opts);
  ...
}
```

or shorter, without an extra identifier, like this:

```
function skipExitTracing(options) {
  options = Object.assign(
    {
      isActive: true,
      extendedResponse: false,
      skipParentSpanCheck: false,
      log: true,
      skipIsTracing: false
    },
    options
  );
  // console.table(options);
   ...
}
```

Reply:
Ah ja good catch 👍 I thought it was the other way around :)